### ISSUE TITLE
running behind nginx reverse proxy

### ISSUE THREAD

**Issue by paulczar** [drone/drone-wall#11]
if I run this behind nginx I only see the text and not the css etc to make it look pretty.

started with `/usr/local/bin/drone-wall --datasource=/var/lib/drone/drone.sqlite --repos=github.com/xxxx/xxxx --port=:8500'

nginx config is dead simple 

```
       location /wall {
                proxy_pass  http://127.0.0.1:8500;
                proxy_redirect     off;
        }
```

viewing it on port 8500 looks fine,  through the nginx proxy:

```
    [[ project.projectName ]]
    [[ project.projectOwner ]]
        [[ build.pull ? 'Pull Request #' + build.pull : build.message ]]
        [[ build.buildTime | date:'M/d, h:mm a' ]]

```

**Comment by mjschultz** [comment]
I can confirm that setting up drone-wall as a top-level route works while setting it up as a route like `/wall` does not work. I would prefer a route like `/wall` to work though!

**Comment by Tathanen** [comment]
Per https://github.com/drone/drone-wall/commit/b4d0c95b1bb8f2f451d8e7388db4ef5ca6d65a7d, can now install in non-root directories, ala `/wall`.  See addition to README.

Will be included in `3.1.0` release.
